### ISSUE TITLE
vfnet loss is so weird, please help.

### ISSUE THREAD

**Issue by yeonsikch** [open-mmlab/mmdetection#8619]
Hi.

I'm learning vfnet on my custom dataset. [config](https://github.com/open-mmlab/mmdetection/blob/master/configs/vfnet/vfnet_x101_64x4d_fpn_mdconv_c3-c5_mstrain_2x_coco.py)

I have 4 GPUs(3080ti*4) and `sample_per_gpu = 1`.
There is one object per image, and the scale is about 1/9 of the image area.
The total number of images is over 200,000.

Any advice would be appreciated.
Thank you.

This is my custom config file:
```
_base_ = '../vfnet/vfnet_x101_64x4d_fpn_mdconv_c3-c5_mstrain_2x_coco.py'

dataset_type = 'CocoDataset'

data_root = 'data/COCO_AI_HUB/'
classes = ('Paper', 'Carton', 'Can', 'Glass', 'Plastic bottle', 'Plastic', 'Pastic bag')
num_classes = 7

model = dict(
    bbox_head=dict(
        num_classes=num_classes
    )
)

data = dict(
    samples_per_gpu=1,
    workers_per_gpu=2,
    train=dict(
        type=dataset_type,
        ann_file=data_root + 'annotations/aihub_train_annotation.json',
        img_prefix=data_root + 'train/',
        classes=classes,
        ),
    val=dict(
        type=dataset_type,
        ann_file=data_root + 'annotations/aihub_val_annotation.json',
        img_prefix=data_root + 'val/',
        classes=classes,
        ),
    test=dict(
        type=dataset_type,
        ann_file=data_root + 'annotations/aihub_val_annotation.json',
        img_prefix=data_root + 'val/',
        classes=classes,
        )
)

log_config = dict(
    interval=10,
    hooks=[
        dict(type='TextLoggerHook'),
        dict(
            type='WandbLoggerHook',
            init_kwargs=dict(
                project='trash_project',
                name='vfnet_aihub')
        )
    ]
)

checkpoint_config = dict(interval=1,out_dir='work_dirs/vfnet_aihub/')

load_from = 'models/vfnet_x101_64x4d_fpn_mdconv_c3-c5_mstrain_2x_coco_20201027pth-b5f6da5e.pth'

```

This is my custom data example:
![15_X001_C014_1016_1](https://user-images.githubusercontent.com/79968598/186094735-4763cf60-101e-4cec-a263-63573dc1a95a.jpg)
![15_X001_C013_1013_2](https://user-images.githubusercontent.com/79968598/186094752-6b370757-3b4f-4a11-805b-ef5e548070d3.jpg)
![15_X001_C013_1013_2](https://user-images.githubusercontent.com/79968598/186094758-fe6094e8-ed5b-4964-a688-b8ecf39feb7c.jpg)

This is my wandb log :
![image](https://user-images.githubusercontent.com/79968598/186090923-569eb4e3-bd88-4a74-92ad-c5c135733886.png)
![image](https://user-images.githubusercontent.com/79968598/186091004-73de28a6-09f6-48ec-ae33-0d079b5dd2ee.png)

**Comment by PeterVennerstrom** [comment]
Looks like vfnet is trained with 8 gpus and 2 samples per gpu for a total batch size of 16.  The learning rate used for batch size 16 is .01.  

You could try reducing your learning rate to .0025 based on your batch size of 4.

**Comment by yeonsikch** [comment]
Thank you! I'll try and re-write result.
